index.html

```diff
@@ -43,6 +43,263 @@
             <img src="assets/Illustration.png" alt="Illustration" class="illustration">
         </div>
     </section>
+    <section class="my_work">
+        <div class="container">
+            <div class="my_recent_work">
+                <h2>My Recent Works</h2>
+                <div class="line">
+                    <img src="assets/Vector (1).png" alt="line">
+                </div>
+                <div class="post_img"></div>
+                <div class="mutli_post"><h2>Multi-Post Stories</h2></div>
+                <div class="txt">
+                    <p>
+                         A daily selection of privately personalized
+                        <br> reads; no accounts or sign-ups required.
+                        <br> has been the industry's standard dummy
+                        <br> text ever since the 1500s, when an
+                        <br> unknown printer took a standard dummy
+                        <br> text.
+                    </p>
+                </div>
+                <div class="projects">
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div> 
+                <div class="c_pro">   
+                    <a href="#">See Project</a> 
+                </div>
+            </div>
+            <div class="posts">
+                <div class="p_title">
+                    <h1>Profesional Art <br>Printing Data</h1>
+                    <p>
+                        A daily selection of privately 
+                        <br>personalized reads; no accounts or
+                        <br>sign-ups required. Has been the 
+                        <br>industry's standard.
+                    </p>
+                    <div class="p_p">
+                        <ul>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
```

Comment:
Great work on implementing this section, but kindly I suggest you remove the link (`a` tag) in the stacks used for each project. 😄